test/index.test.ts

```diff
@@ -0,0 +1,7 @@
+import { promises as fs } from 'fs';
+
+describe('the application', (): void => {
+  it('should exist', async (): Promise<void> => {
+    await expect(fs.access(`${__dirname}/../src/index.ts`)).resolves.not.toThrow();
```

Comment:
reasonable placeholder. Assumes that is present even after its compilation, which is probably what happens in `dev`.